packages/global/components/layouts/content/default.marko

```diff
@@ -76,7 +76,7 @@ $ const shouldInjectAds = ["article", "blog", "news", "podcast", "press-release"
               required-access-level-ids=accessLevels
             >
               <if(!context.canAccess || context.requiresUserInput)>
-                $ const body = getContentPreview({ body: content.body, selector: "p:lt(3)" });
+                $ const body = getContentPreview({ body: content.body, selector: "p:lt(5)" });
```

Comment:
If this is supposed to appear after the 5th paragraph this should be `p:lt(6)` as `p:lt(3)` is after the second paragraph.